stdlib/3/unittest/mock.pyi

```diff
@@ -92,7 +92,11 @@ if sys.version_info >= (3, 3):
         def start(self) -> Any: ...
         def stop(self) -> Any: ...
 
-    def patch(target: Any, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> Any: ...
+    class _patcher:
+        def __call__(self, target: Any, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> Any: ...
+        def object(self, target: Any, attribute: str, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> _patch: ...
```

Comment:
Thanks for the feedback @matthiaskramm and @gvanrossum !  I wound up using `None` to be consistent with the other call sites with a similar signature in this file.